components/SignupSection/SignupSection.jsx

```
@@ -618,6 +624,20 @@ const SignupSection = () => {
+          <div className={styles.checkBoxRow}>
+            <Checkbox
+              checked={acceptedMarketingConsent}
+              onChange={handleMarketingConsentCheckbox}
+              sx={{
+                "& .MuiSvgIcon-root": {
+                  color: "#D1151A",
+                },
+              }}
+            />
+            <span style={{ fontSize: "0.7rem" }}>
+              Eu aceito receber comunicações de marketing, promoções e ofertas personalizadas do Hemocione e seus parceiros.
```

ESLint

[error] Insert `⏎·············`

(prettier/prettier)